tools/translate_annotations.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
```

Comment:
Nit:  would be nice to add short description of what this script is about

Reply:
Ah thanks! I meant to do this in `tools/README.md` but somehow I forgot; I'll add that